### PR TITLE
fix(hypha-wallet): pop-up content

### PR DESCRIPTION
**Pull request by evgenibir**
### ✅ Checklist

- Fixed hypha-wallet pop-up content

### CODE REVIEW

File: src/boot/api.js [hunk 1]
```diff
@@ -16,14 +17,27 @@ const signTransaction = async function (actions) {
   let error = null
   try {
     if (this.$type === 'ual') {
+      let options = {}
+      if (Screen.gt.md) {
+        options = {
+          blocksBehind: 3,
+          expireSeconds: 30,
+          text: 'Scan the QR-code with Hypha Wallet on your mobile device in order to sign this transaction request',
```
Comment (Alex-IS) [inline src/boot/api.js:25]: @evgenibir this text should be translated

File: src/boot/api.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { Api, JsonRpc } from 'eosjs'
 import axios from 'axios'
+import { Screen } from 'quasar'
 
 const signTransaction = async function (actions) {
```
Comment (Alex-IS) [inline src/boot/api.js:5]: @evgenibir signTransaction should accept `options` as second parameter so it could be easily overwritten if needed

**Review by Alex-IS — CHANGES_REQUESTED** [review]

File: src/boot/api.js (hunk 1 above)
Comment (evgenibir) [reply src/boot/api.js]: fixed

File: src/boot/api.js (hunk 2 above)
Comment (evgenibir) [reply src/boot/api.js]: fixed

File: src/store/accounts/actions.js [hunk 3]
```diff
@@ -219,7 +222,21 @@ export const verifyOTP = async function ({ commit, dispatch, state, rootState },
     }
   })
 
-  await this.$api.signTransaction(actions)
+  let options = {}
```
Comment (Alex-IS) [inline src/store/accounts/actions.js:225]: @evgenibir we should have this as default value in one place and then reuse it instead of copy / pasting it

**Review by Alex-IS — CHANGES_REQUESTED** [review]

File: src/boot/api.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import { Api, JsonRpc } from 'eosjs'
 import axios from 'axios'
 
-const signTransaction = async function (actions) {
+const signTransaction = async function (actions, options) {
```
Comment (Alex-IS) [inline src/boot/api.js:4]: @evgenibir  so here you could change to `async function (actions, options = DEFAULT_OPTIONS )` or where DEFAULT_OPTIONS is the value that you've been copy / pasting

**Review by Alex-IS — CHANGES_REQUESTED** [review]

File: src/store/accounts/actions.js (hunk 3 above)
Comment (evgenibir) [reply src/store/accounts/actions.js]: fixed

File: src/boot/api.js (hunk 4 above)
Comment (evgenibir) [reply src/boot/api.js]: fixed

**Review by Alex-IS — APPROVED** [review]

**Pull request merged**